### PR TITLE
Add a save button and account summary to plan page

### PR DESCRIPTION
**Pull request by cvializ**
@JRuff42 @andrewconner 
screen shot 2015-10-16 at 3 54 29 pm

### CODE REVIEW

**Comment by cvializ** [comment]
( @camhashemi )

File: kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    return {
+      restrict: 'A',
+      replace: true,
+      templateUrl: 'teamSettings/billingSummary.tpl.html',
+      scope: {
+        billingState: '='
+      },
+      link: function ($scope) {
+        $scope.isPaidPlan = function () {
+          return $scope.billingState.plan.pricePerUser !== '$0.00'; // TODO: i18n will break this
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.js:16]: I'm going to add `.isFree` to each `plan` so we don't have to do this all over the place. The price per user doesn't matter, what matter is if we want them to upgrade or not.

File: kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.tpl.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        As of today, your next bill will be
+        <b ng-bind="billingState.charge"></b>
+        on
+        <b ng-bind="billingState.billingDate | date:'yyyy-MM-dd'"></b>.
+      </p>
+    </li>
+    <li>
+      <p>
+        <span>You have a&nbsp;</span>
+        <u ng-bind="'credit of ' + billingState.balance"></u>
```
Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.tpl.html:23]: When `billingState.balance` is negative, this wouldn't make sense

File: kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.js (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.js:16]: kk, I'll update it when you add .isFree

File: kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.tpl.html (hunk 2 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/teamSettings/billingSummary.tpl.html:23]: nice catch, I have a commit ready for this one, but I need to make a PR

**Pull request merged**